ubuntu_image/helpers.py

```diff
@@ -5,8 +5,9 @@
 import logging
 
 from contextlib import ExitStack, contextmanager
+from parted import Device
 from subprocess import PIPE, run as subprocess_run
-from tempfile import TemporaryDirectory
+from tempfile import TemporaryDirectory, mkstemp
```

Comment:
Oh my, yes! Too much old python 2 in the past, was just recently working on something that had to support all python versions and without thinking just used mkstemp. Fixing! Thanks!